### PR TITLE
Fix component UI kit and added variants to them

### PR DESCRIPTION
**Pull request by mikawix**
1. Fixed price issue when price is not defined
image
(showed price with a strike through before change)

2. Added missing variants to components
3. Added label variant
4. Added animation kit - variants to animations board, but Parallax, need your help on it

### CODE REVIEW

File: _codux/ui-kit/components/components.board.tsx [hunk 1]
```diff
@@ -41,21 +49,26 @@ export default createBoard({
...
                 </Variant>
+                <hr className={styles.hrLight} />
                 <p className={classNames(styles.variantName, styles.headlinesSpacing)}></p>
                 <h4 className={styles.sectionHeader}>LABELS</h4>
-                <h4>Labels missing</h4>
+                <div className="relativeContainer">
+                    <Variant name="Label">
+                        <span className={productCardStyles.ribbon}>I&apos;M A LABEL</span>
+                    </Variant>
+                </div>
```
Comment (alisey) [inline _codux/ui-kit/components/components.board.tsx:60]: This is not a reusable class, it's an internal part of the product card component, don't put it here.

File: src/styles/common.scss [hunk 2]
```diff
@@ -365,3 +365,8 @@
 .alternateBackground {
     background-color: var(--colorA3);
 }
+.relativeContainer {
+    position: relative;
+    width: 100%;
+    height: 50px;
+}
```
Comment (alisey) [inline src/styles/common.scss:372]: No.

File: src/components/product-price/product-price.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ interface ProductPriceProps {
 }
 
 export const ProductPrice = ({ price, discountedPrice, className }: ProductPriceProps) => {
-    const hasDiscount = price !== discountedPrice;
+    const hasDiscount = discountedPrice && price !== discountedPrice;
```
Comment (tetianaman) [inline src/components/product-price/product-price.tsx:11]: This is already in `main`

File: src/styles/common.scss (hunk 2 above)
Comment (mikawix) [reply src/styles/common.scss:372]: updated

File: _codux/ui-kit/components/components.board.tsx (hunk 1 above)
Comment (mikawix) [reply _codux/ui-kit/components/components.board.tsx]: updated

**Comment by alisey** [comment]
Implemented in:
https://github.com/codux-templates/e-commerce-reclaim/pull/112
https://github.com/codux-templates/e-commerce-reclaim/pull/111

**Pull request closed without merging**